piet-common/src/direct2d_back.rs

```diff
@@ -21,7 +27,7 @@ pub type Piet<'a> = D2DRenderContext<'a>;
 /// The associated brush type for this backend.
 ///
 /// This type matches `RenderContext::Brush`
-pub type Brush = GenericBrush;
+//pub type Brush = GenericBrush;
```

Comment:
The doc comments will apply to the next item; delete.

Reply:
I forgot why I commented out `Brush` and `Bitmap`; I think it was a temporary thing to get stuff to compile. Perhaps when we did a rebase?

In any case, on closer inspection, I can export `Brush` and `Bitmap` from `piet_direct2d`, would that be preferred?

Reply:
I *think* the answer here is that they're exported as associated types of `RenderContext`, and that should be considered their "true names" for the purpose of other code using them. Otherwise they can be considered internal details. So I think the right thing to do is delete them.

We will probably come back to the question of exporting platform-specific capabilities (especially the ability to create D2D compatible images from other sources than memory buffers). But I don't think that just exporting the `Brush` or `Bitmap` type helps us all that much.

Reply:
Ok, Brush and Bitmap deleted.